tests/test_wave_01.py

```diff
@@ -30,9 +29,13 @@ def test_get_tasks_one_saved_tasks(client, one_task):
             "is_complete": False
         }
     ]
+    new_task = Task.query.get(1)
```

Comment:
In this case, accessing the DB directly is once again testing the test set-up, not the user path. So these asserts don't add any benefit to the asserts that came before it, which is directly testing the _user_ pathway of the code by checking the response, which is the more relevant part to test.

That is, testing the test set-up doesn't provide you with any new information about if the code works, just that your test set-up worked. Which can be useful to verify when you're getting that set-up set up, but is probably not in this case where we're more interested in the return value of the `GET /tasks` endpoint.

Reply:
I can already see a few other cases, but I'll stop pointing them out to not repeat myself.